packages/manager/modules/vps/src/dashboard/vps-dashboard.html

```diff
@@ -1,5 +1,10 @@
 <ovh-manager-incident-banner service-name="$ctrl.serviceName">
 </ovh-manager-incident-banner>
+
+<vps-announcement-banner
+    data-can-display-banner="$ctrl.canDisplayVpsAnnouncementBanner">
```

Comment:
Why not directly set the `ng-if` on this element ? This will avoid to pass the method to the component, and use it on its main HTML element